exporters/common/src/main/java/io/opentelemetry/exporter/internal/InstrumentationUtil.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.internal;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.ContextKey;
+import java.util.Objects;
+
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+public class InstrumentationUtil {
+  private static final ContextKey<Boolean> SUPPRESS_INSTRUMENTATION_KEY =
+      ContextKey.named("suppress_instrumentation");
```

Comment:
Let's name this something unlikely to collide with any future general spec around this. Something like `suppress_internal_exporter_instrumentation`?

Reply:
(it won't technically collide since context keys are identity-based, but 👍)

Reply:
Thanks, it's updated now